src/vastRequest.coffee

```diff
@@ -0,0 +1,52 @@
+class VastRequest
+  constructor: (@url, @eventEmitter, @callback) ->
+    @clickThrough = undefined
+    @run() if @url
+
+  valid: ->
+    return false unless @url
+    return true
+
+  run: -> @requestHandler(@setTrackings)
+
+  setTrackings: (xmlDomTree) =>
+    return unless xmlDomTree
+    @xmlDomTree = xmlDomTree # TODO: debug, delete
+    trackings = []
+    @clickThrough = xmlDomTree.querySelector("ClickThrough").textContent
+    Array.prototype.forEach.call(
+      xmlDomTree.querySelectorAll("Tracking"),
```

Comment:
Do you plan to support just `<Linear>` elements? It might be a good idea to increase the specificity of the selector, e.g.

```
xmlDomTree.querySelectorAll("Linear TrackingEvents Tracking")
```

VAST allows multiple creatives. How will you ensure that this doesn't pick up the tracking details of different creatives? Perhaps `xmlDomTree` is guaranteed to be a subtree for a singe creative?